### DISCUSSION TITLE
Sonic Generations has the original game on genesis. But no one to dare extract it.

### DISCUSSION THREAD

**Discussion by Gabrielx36** [farmerbb/RED-Project#66] [Ideas]
Hello, In case anyone forgot. When you beat Sonic generations, you get a genesis controller and can play the original sonic the hedgehog. This got me thinking, how to extract the rom from the game. Please let me know of you can figure it out. Also, remember, Sonic Generations came on xbox 360 ps3 and pc. So different methods will be required.

**Comment by TheNametag** [comment]
It's not in the PC version.  Even unpacking the files, I don't see the ROM remaining anywhere.

**Comment by Gabrielx36** [comment]
Man does Sega like to keep things a secret. But I do remember playing Sonic 1 after completing Sonic Generations on my Xbox 360. I wonder if Sega hid it using a different file extension.

**Reply by Gabrielx36** [reply]
Could you look for a sus looking .bin file
